public/data/cchow33.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Carmen Chow",
+  "type": "personal",
+  "bio": "Aspiring FE Dev!",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/cchow33",
+      "icon": "github"
+    },
```

Comment:
There's an extra comma in here.
```suggestion
    }
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable